canonical-websites/snapcraft.io
# Update ES, set asset version number as variable

Updated Spanish Instala to Instalar as per https://github.com/canonical-websites/snapcraft.io/commit/e7ab4d663cb6f927f75b8f697a8be93a1cf4af4b#r31297033.

Also set the download version number to be a variable for easier updating in the future.

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/<snap>/publicise
- Ensure the Spanish version has the correct text, both in the copy and the image
